### PR TITLE
chore: added docker-compose to readme.md

### PR DESCRIPTION
**Pull request by bluecitylights**
Added example of docker-compose for easier handling the docker instance

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +244,29 @@ Generatr container. So
 is needed to expose the container's port 8080 to the host (web browser). In the example above, the
 `-p 8080:8080` argument tells Docker to bind the local machine / host's port 8080 to the container's port 8080.
 
+#### Simplify using Docker-Compose
+Using Docker-Compose we can simplify the process.
```
Comment (dirkgroot) [inline README.md:248]: Docker Compose is written without the hyphen. Also, a link to the Docker Compose site would be nice.

```suggestion
#### Simplify using Docker Compose
Using [Docker Compose](https://docs.docker.com/compose) we can simplify the process.
```

File: README.md [hunk 2]
```diff
@@ -243,6 +244,29 @@ Generatr container. So
...
+Add a docker-compose.yml next to your workspace.dsl.
+Add contents
+```
+version: '3'
+services:
+  structurizr-site-generatr:
+    image: ghcr.io/avisi-cloud/structurizr-site-generatr
+    volumes:
+      - .:/var/model
+    # command: generate-site -w workspace.dsl
```
Comment (dirkgroot) [inline README.md:258]: This line can be removed.

**Review by dirkgroot — CHANGES_REQUESTED** [review]
Thanks for this useful addition! I've added 2 remarks.

File: README.md (hunk 2 above)
Comment (qtzar) [reply README.md:258]: I'd also maybe make it clear in the readme that you can edit the Docker Compose file to adjust the generate-site command with other switches (like adding the assets folder) and adjust the port where needed.

File: README.md [hunk 3]
```diff
@@ -243,6 +244,30 @@ Generatr container. So
 is needed to expose the container's port 8080 to the host (web browser). In the example above, the
 `-p 8080:8080` argument tells Docker to bind the local machine / host's port 8080 to the container's port 8080.
 
+#### Simplify using Docker Compose
```
Comment (dirkgroot) [inline README.md:247]: Please add this chapter to the table of contents.

**Review by dirkgroot — CHANGES_REQUESTED** [review]
Looks good, thanks! I've added one last additional remark.

**Comment by jp7677** [comment]
An alternative (my preference) for including this in the readme would be an entry here https://github.com/avisi-cloud/structurizr-site-generatr/discussions/categories/q-a

**Comment by jp7677** [comment]
As stated here https://github.com/avisi-cloud/structurizr-site-generatr/pull/304#issuecomment-1772214674 let’s also move this instructions to a separate markdown file in the docs folder. We can later refer to this from the q-a and also from the readme.

**Comment by dirkgroot** [comment]
I'm closing this PR for now. Feel free to reopen it if you want to continue working on this.

**Pull request closed without merging**